proxy/envoy/config.go

```diff
@@ -316,7 +321,7 @@ func buildOutboundRoutes(instances []*model.ServiceInstance, services []*model.S
 				http.VirtualHosts = append(http.VirtualHosts, host)
 
 			case model.ProtocolTCP, model.ProtocolHTTPS:
-				cluster := buildOutboundCluster(service.Hostname, port, nil)
```

Comment:
Please do not enable auth for TCP unless you properly thought about the design implications.